mobile-app/app/screens/RootNavigator.tsx

```diff
@@ -16,6 +17,10 @@ export function RootNavigator (): JSX.Element {
     isLoaded
   } = useWalletPersistenceContext()
 
+  useEffect(() => {
+    void WalletNotifications.register()
+  }, [])
```

Comment:
this should be part of settings and asked upon wallet creation since you are asking for permission, should not register every time?

Reply:
With the register function we are first fetching notification permission, and if the status is `GRANTED` then we are not requesting it again but just updating the [status](https://github.com/DeFiCh/wallet/blob/a6ce2394b345a3df3234a0320e4d4940f5c02078/mobile-app/app/api/wallet/notifications.ts#L15) variable for future purpose

Reply:
If we set notifs as false by default, can we just call it when we enable it?